test/Sentry.Testing/TestOutputDiagnosticLogger.cs

```diff
@@ -18,6 +18,8 @@ public class LogEntry
         public SentryLevel Level { get; set; }
         public string Message { get; set; }
         public Exception Exception { get; set; }
+        public object[] Arguments { get; set; }
```

Comment:
Arguments got added but it's not used by any test?

Reply:
good catch.  will fix